### DISCUSSION TITLE
Add SurrealDB support

### DISCUSSION THREAD

**Discussion by ibilux** [mikro-orm/mikro-orm#5822] [Ideas]
**Describe the solution you'd like**
I would highly appreciate if [SurrealDB](https://github.com/surrealdb/surrealdb) could be a supported DBMS.

**Comment by ibilux** [comment]
There is already a `Kysely` dialects for SurrealDB at: [kysely-surrealdb](https://github.com/igalklebanov/kysely-surrealdb)

**Reply by B4nan** [reply]
This is not really relevant, we dont use kysely here (maybe we'll do it at some point, but that will be a huge effort on its own).

**Comment by B4nan** [comment]
I am personally not interested in supporting too many dialects inside the repository itself, such dialects should be created and published as 3rd party modules preferably. The ORM is already extensible, you can create your own drivers.

**Comment by ibilux** [comment]
@B4nan I mentioned the `Kysely` dialects as a reference only (as they mentioned that [SurrealQL](https://surrealdb.com/docs/surrealql) is based on SQL, so why not?), And I'm not suggesting using it.
I love how `mikro-orm` uses `decorators`  and that's why I would appreciate if  `mikro-orm` can support SurrealDB.
